bridge/core/auth/credential.go

```diff
@@ -32,17 +32,23 @@ func NewErrMultipleMatchCredential(matching []entity.Id) *entity.ErrMultipleMatc
 	return entity.NewErrMultipleMatch("credential", matching)
 }
 
+// Special Id to mark a credential as being associated to the default user, whoever it might be.
+// The intended use is for the bridge configuration, to be able to create and store a credential
+// with no identities created yet, and then select one with `git-bug user adopt`
+const DefaultUserId = entity.Id("default-user")
+
 type Credential interface {
 	ID() entity.Id
 	UserId() entity.Id
+	updateUserId(id entity.Id)
```

Comment:
I changed it for a private method because we don't need it exposed at the moment and it prevent an incorrect usage. If it needs to change later we can.